pom.xml

```diff
@@ -21,8 +21,8 @@
     <build-helper-maven-plugin.version>3.2.0</build-helper-maven-plugin.version>
     <maven-clean-plugin.version>2.5</maven-clean-plugin.version>
     <maven-assembly-plugin.version>3.3.0</maven-assembly-plugin.version>
-    <frontend-maven-plugin.version>1.9.1</frontend-maven-plugin.version>
-    <node.version>14.14.0</node.version>
+    <frontend-maven-plugin.version>1.12.0</frontend-maven-plugin.version>
+    <node.version>16.1.0</node.version>
```

Comment:
I believe 16.2.0 is out now. May be good to update it while we're here.

Reply:
It may be worth noting that Node v16 will be the **next** LTS ([ref](https://nodejs.org/en/about/releases/)) starting in October, so it's the best version to jump to since we wish to get a compatibility feature not offered in the current LTS.

Reply:
@ivan-gomes, thank you. I updated node.js to 16.2.0